src/Cubie/Corners.hs

```diff
@@ -0,0 +1,20 @@
+module Cubie.Corners where
+
+import Prelude hiding (length)
+import Cubie.Misc
+import Data.Maybe
```

Comment:
Is this also import automatically?

What is the standard for this sort of thing? Should it always be imported even if implied?